### ISSUE TITLE
Mapa de navegação

### ISSUE THREAD

**Issue by huogerac** [huogerac/todolist-turma1#7]
# Visão geral

Embora não precisamos fazer o protótipo de todas as páginas no nosso sistema de gestão de tarefas, páginas de login, listagem, cadastrar, alterar etc..
Devemos pensar na navegação entre estas páginas e quais páginas devem existir! Qual será a página principal? Quando o usuáro faz login, o que acontece?

# Tarefa

👉 Fazer um desenho (também usando papel e lápis) das páginas principais e sua navegação, ou seja, Página A --> Página B (utilizando setas apontando origem e destino)

**Comment by huogerac** [comment]
Independente do conteúdo de cada página, abaixo segue uma ideia de fluxo de navegação.

Ao fazer o protótipo de cada página, conforme a tarefa #6 , esta navegação pode ser alterada:

![mapeamento-navegacao](https://user-images.githubusercontent.com/962233/157126804-33cdc7dd-6a6c-44e7-a380-f6e3dac06074.jpg)

### TODO

- Adicionar navegação do fluxo de login/logout

**Comment by julianapereiradev** [comment]
Oii gente, boa noite!
Tentei juntar um pouco do que o professor fez + o que o pessoal do grupo colocou no post anterior e ficou mais ou menos assim:
![navegacao_1](https://user-images.githubusercontent.com/100153263/157337898-e86d2367-0929-4712-a2a8-0b5567a307d8.JPG)

![navegacao_2](https://user-images.githubusercontent.com/100153263/157337918-89dcebe3-65af-4d65-9901-045e309043b2.JPG)

**Comment by marcelalpg** [comment]
Minha sugestão de navegação entre as páginas juntando as ideias dos outros protótipos. 
![20220312_063510](https://user-images.githubusercontent.com/100198220/158064867-1c1a3459-3ac2-4c5a-a1c4-f30fb70214d4.jpg)

**Comment by marcelalpg** [comment]
Desculpem-me ficou faltando uma parte. 
![20220315_064629](https://user-images.githubusercontent.com/100198220/158352493-d06c444b-c0bd-4afe-be84-061211277feb.jpg)

**Comment by huogerac** [comment]
@marcelalpg uma coisa que gostei bastante eh o conceito de progresso das tarefas que voce colocou no seu prototipo...uma sacada bem legal, que nao vi em nenhum todo list

**Comment by huogerac** [comment]
Fala pessoal,

Gostaria de fechar o quanto antes esta issue aqui, e pegando as sugestoes de voces, acabei chegando neste mapa aqui em baixo

## mapa de navegacao (resumo)
- A ideia eh conseguir validar as possibilidades de navegacao entre paginas e quais paginas existem.
- Entendo que temos 2 conjuntos de paginas
  - Paginas referente ao login autenticacao (login, cadastro de usuario)
  - Paginas referente as tarefas
  
![image](https://user-images.githubusercontent.com/962233/160194150-dc0dfc08-2980-4ec4-a1c8-de0cecaf66d6.png)

## Funcionalidades  - Paginas

- 1# Landing page da nossa app (home, pagina principal, index.html)
- 2# Pagina de Login (login, signin)
- 2.1# Cadastrar novo usuario (register, signup)
- 3# Painel de tarefas (painel, dashboard, summary)
- 4# **Pagina de listagem** (prototipo fechado #6 )
- 4.1# Nova tarefa
- 4.2# Alterar tarefa
- 4.3# Remover tarefa
- 5# Novo grupo
- 5.1# Alterar grupo
- 5.2# Excluir grupo

**Notas**
- As paginas nao tem muitos detalhes no mapa, o importante eh saber: i) as paginas existentes ii) as possibilidades de navegacao entre paginas
- As paginas referente ao login de usuario, podem ficar mais completas, com integracao com google/facebook, reset de senha etc...MAS acredito que i) todo lugar tem este fluxo ii) podemos manter simples e focar no negocio que eh o todo list. Ou seja, fazer o mais simples possivel, depois se sobrar tempo, podemos gastar tempo discutindo a parte de login/cadastro de usuarios, o que acham?
- Estao sentindo falta de alguma pagina?
- Conseguem ver algum problema na navegacao?

**Comment by julianapereiradev** [comment]
> Fala pessoal,
> 
> Gostaria de fechar o quanto antes esta issue aqui, e pegando as sugestoes de voces, acabei chegando neste mapa aqui em baixo
> 
> ## mapa de navegacao (resumo)
> * A ideia eh conseguir validar as possibilidades de navegacao entre paginas e quais paginas existem.
> * Entendo que temos 2 conjuntos de paginas
>   
>   * Paginas referente ao login autenticacao (login, cadastro de usuario)
>   * Paginas referente as tarefas
> 
> ![image](https://user-images.githubusercontent.com/962233/160194150-dc0dfc08-2980-4ec4-a1c8-de0cecaf66d6.png)
> 
> ## Funcionalidades - Paginas
> * 1# Landing page da nossa app (home, pagina principal, index.html)
> * 2# Pagina de Login (login, signin)
> * 2.1# Cadastrar novo usuario (register, signup)
> * 3# Painel de tarefas (painel, dashboard, summary)
> * 4# **Pagina de listagem** (prototipo fechado [Protótipo & Conteúdo (Wireframes) #6](https://github.com/huogerac/todolist-turma1/issues/6) )
> * 4.1# Nova tarefa
> * 4.2# Alterar tarefa
> * 4.3# Remover tarefa
> * 5# Novo grupo
> * 5.1# Alterar grupo
> * 5.2# Excluir grupo
> 
> **Notas**
> 
> * As paginas nao tem muitos detalhes no mapa, o importante eh saber: i) as paginas existentes ii) as possibilidades de navegacao entre paginas
> * As paginas referente ao login de usuario, podem ficar mais completas, com integracao com google/facebook, reset de senha etc...MAS acredito que i) todo lugar tem este fluxo ii) podemos manter simples e focar no negocio que eh o todo list. Ou seja, fazer o mais simples possivel, depois se sobrar tempo, podemos gastar tempo discutindo a parte de login/cadastro de usuarios, o que acham?
> * Estao sentindo falta de alguma pagina?
> * Conseguem ver algum problema na navegacao?

Concordo totalmente! Primeiro seria legal a gente focar em criar as paginas html/css do listar.html (que a gente já começou na monitoria) + as outras que estão faltando. Sempre tentando colocar as funcionalidades principais e, só depois, ver o que a gente pode incrementar no todolist
